daisi/src/path_planning/time_calculation_helper.cpp

```diff
@@ -81,13 +80,10 @@ static double estimatesCostsInTime(const ns3::Vector &start, const ns3::Vector &
   double max_acceleration = kinematics.getMaxAcceleration();
   double min_acceleration = kinematics.getMaxDeceleration();
 
-  double distance_acc;
-  double distance_brake;
-  double d_const;
-
   // calculate distance threshold
-  distance_acc = (max_velocity * max_velocity) / (2 * (max_acceleration));
-  distance_brake = (max_velocity * max_velocity) / (2 * std::abs(min_acceleration));
+  double distance_acc = (max_velocity * max_velocity) / (2 * (max_acceleration));
+  double d_const = 0.0;
```

Comment:
The declaration of d_const could be moved further down to where it is defined with another value and used, since it is never used with this default value